src/main/java/calculator/CalculatorMain.java

```diff
@@ -0,0 +1,10 @@
+package calculator;
+
+import static view.InputView.getInput;
+
+public class CalculatorMain {
+
+    public static void main(String[] args) {
+        getInput();
```

Comment:
```suggestion
        String text = InputView.inputText();
        int result = Calculator.calculate(text);
        OutputView.printResult(result);
```
이와 같이 main이 모델과 뷰의 controller 역할을 하는 방식으로 구현할 것을 추천
지금 구현은 View가 모델을 호출하고, 모델에서 View로 값을 전달하는 구조임
모델과 뷰 사이에 중간자 역할을 하는 Controller 추가할 것을 추천

Reply:
헉 이해했습니다.. 감사합니다!!